src/main/features/claim/views/incomplete-submission.njk

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.njk" %}
+
+{% set title = t('Incomplete submission') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('You need to complete all sections before you submit your claim') }}</h1>
```

Comment:
Lines 8-17 could go into `cmc-commons-frontend` as compost element. The fragment could look something like this:

```
{{ remainingTasksFragment(heading, tasks, taskListLink) }}
````

I would like us share not only low level macros like form controls but also more complex  page fragments.

Reply:
done